.gitlab-ci.yml

```diff
@@ -269,10 +290,14 @@ regression-include-excluded-packages:
     # BLACKLIST: Skipped on fedora systems
     - if: $RUNNER !~ "/^.*(fedora).*$/" && $CI_PIPELINE_SOURCE != "schedule"
     - !reference [.nightly_rules_all, rules]
+    - !reference [.ga/eus_rules_all, rules]
+    - !reference [.ga/eus_rules_exclude, rules]
   variables:
     SCRIPT: regression-include-excluded-packages.sh
 
 regression-old-worker-new-composer:
+  rules:
+    - !reference [.ga/eus_rules_all, rules]
```

Comment:
Missing `    - !reference [.upstream_rules_all, rules]` ? or rather shouldn't this be the same as other `regression-` jobs ?